spec/board_spec.rb

```diff
@@ -0,0 +1,62 @@
+require './lib/board' 
+
+RSpec.describe Board do
+  describe "initialize" do
+    it "returns an array with nil cells value" do
+      board = Board.new(3)
+      expect(board.matrix).to eql([nil,nil,nil,nil,nil,nil,nil,nil,nil])
```

Comment:
Why do you need to test this? As a rule of thumb, test the `public` methods. The way to test is: Am I receiving what I expected to receive?